easybuild/easyconfigs/n/NVHPC/NVHPC-20.7.eb

```diff
@@ -6,10 +6,9 @@ description = """C, C++ and Fortran compilers included with the NVIDIA HPC SDK (
 
 toolchain = SYSTEM
 
-# NVHPC can be downloaded freely from NVIDIA's website at https://developer.nvidia.com/hpc-sdk
-# It requires accepting the HPC SDK Software License Agreement (https://docs.nvidia.com/hpc-sdk/eula/index.html)
-# If you accept the License Agreement, you may also uncomment the following line to automatically download the sources
-# source_urls = ['https://developer.download.nvidia.com/hpc-sdk/%(version)s/']
+# By downloading, you accept the HPC SDK Software License Agreement (https://docs.nvidia.com/hpc-sdk/eula/index.html)
+accept_eula = True
```

Comment:
@AndiH I don't think we should be accepting the EULA automatically...

Reply:
We have the OK from NVIDIA to do that, so I think it's fine. See here: https://github.com/easybuilders/easybuild-easyconfigs/pull/11920#issue-543534877

Reply:
Well, I disagree. What if the person installing NVHPC with EasyBuild doesn't realize they're accepting the EULA?

You can configure EasyBuild to accept specific EULAs (via `--accept-for=EULA`, or by setting `$EASYBUILD_ACCEPT_FOR=EULA`, etc.), or blindly accept any EULA if you want to (`--accept-for=*`), but we should never do that on behalf of the user by default...

So we should drop the `accept_eula = True` from the easyconfig where we do this already, rather than adding it...

Reply:
@boegel I agree with the opt-in approach to accept the NVHPC EULA. Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/12014/commits/a4938e92ce9ad51a54a429daa19831e28d0f5947

Reply:
As you wish. I have nothing against it :).